pyspeckit/spectrum/units.py

```diff
@@ -968,6 +834,27 @@ def _update_from(self, obj):
                 'velocity_convention', 'wcshead', 'xtype'):
             self.__dict__[attr] = obj.__dict__[attr]
 
+    def find_equivalencies(self, velocity_convention, center_frequency, center_frequency_unit, equivalencies):
```

Comment:
These should all be `key=value` arguments.  They should probably default to `None`.